sphinxcontrib/domaintools/__init__.py

```diff
@@ -76,7 +76,7 @@ def add_target_and_index(self, name, sig, signode):
                 indextype = 'single'
                 indexentry = self.indextemplate % (name,)
             self.indexnode['entries'].append((indextype, indexentry,
```

Comment:
Should we add ``if version_info < (1, 4):`` here to also support old Sphinx version? (This also needs import of ``version_info``, see e.g. https://github.com/VACUMM/sphinx-fortran/pull/13/commits/f723ef23480bd27597ecad8d1ad9cd52fb36eb5f)

Reply:
I don't have an opinion about this. Either supporting and not supporting is okay for me.
If we choose to raise the target version of Sphinx, we have to update dependencies on setup.py.

@eric-wieser Could you check this please?